tests/fixtures/responses/describe_database_stats.json

```diff
@@ -1,5 +1,5 @@
 {
     "{\"database_id\": \"kitchen-sink\"}": {
-        "rowCount": 127
+        "rowCount": 0
```

Comment:
This correct?

Reply:
that is weird, isn't it? i don't think it's used anywhere